assets/css/tasks/_contact-tasks.css

```diff
@@ -0,0 +1,98 @@
+/* CONTACT PAGE */
+
+/* assets/css/pages/_contact.css */
+.contact-section {
+    background-image: url(/img/contact-img.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: right 0rem bottom 0;
+   
+}
+.u-center-text {
+    text-align: center;
+}
+.form {
+    /* padding;  set the same space around the contents of the box */
+    padding: 1rem;
+    /* width; the element should occupy the half of the available width */
```

Comment:
Please delete comments if you apply values to properties. They aren't useful because doesn't bring more information, because there are in you code ;)

Reply:
Apply this note to all places.